cocos/spine/lib/spine-define.ts

```diff
@@ -0,0 +1,1555 @@
+    spine.DrawOrderTimeline = wasm.DrawOrderTimeline;
+    spine.IkConstraintTimeline = wasm.IkConstraintTimeline;
+    spine.TransformConstraintTimeline = wasm.TransformConstraintTimeline;
+    spine.PathConstraintPositionTimeline = wasm.PathConstraintPositionTimeline;
+    spine.PathConstraintMixTimeline = wasm.PathConstraintMixTimeline;
+    spine.TrackEntry = wasm.TrackEntry;
+    spine.AnimationStateData = wasm.AnimationStateData;
+    spine.AnimationState = wasm.AnimationState;
+    spine.Animation = wasm.Animation;
+    spine.EventQueue = wasm.EventQueue;
+    //spine.AnimationStateListener = wasm.AnimationStateListener;
+    spine.AnimationStateAdapter = wasm.AnimationStateAdapter;
+    spine.Skeleton = wasm.Skeleton;
+    spine.SkeletonBinary = wasm.SkeletonBinary;
+    spine.SkeletonJson = wasm.SkeletonJson;
+    spine.VertexEffect = wasm.VertexEffect;
+    spine.JitterEffect = wasm.JitterEffect;
+    spine.SwirlEffect = wasm.SwirlEffect;
+
+    spine.SkeletonInstance = wasm.SkeletonInstance;
+}
+
+function overrideProperty_BoneData () {
+    const prototype = spine.BoneData.prototype as any;
+    const propertyPolyfills = [
+        {
+            proto: prototype,
+            property: 'index',
+            getter: prototype.getIndex,
+        },
+        {
+            proto: prototype,
+            property: 'name',
+            getter: prototype.getName,
+        },
+        {
+            proto: prototype,
+            property: 'parent',
+            getter: prototype.getParent,
+        },
+        {
+            proto: prototype,
+            property: 'length',
+            getter: prototype.getLength,
+        },
+        {
+            proto: prototype,
+            property: 'x',
+            getter: prototype.getX,
+        },
+        {
+            proto: prototype,
+            property: 'y',
+            getter: prototype.getY,
+        },
+        {
+            proto: prototype,
+            property: 'rotation',
+            getter: prototype.getRotation,
+        },
+        {
+            proto: prototype,
+            property: 'scaleX',
+            getter: prototype.getScaleX,
+        },
+        {
+            proto: prototype,
+            property: 'scaleY',
+            getter: prototype.getScaleY,
+        },
+        {
+            proto: prototype,
+            property: 'shearX',
+            getter: prototype.getShearX,
+        },
+        {
+            proto: prototype,
+            property: 'shearY',
+            getter: prototype.getShearY,
+        },
+        {
+            proto: prototype,
+            property: 'transformMode',
+            getter: prototype.getTransformMode,
+        },
+        {
+            proto: prototype,
+            property: 'skinRequired',
+            getter: prototype.getSkinRequired,
+        },
+        // {
+        //     proto: spine.BoneData.prototype,
+        //     property: 'color',
+        //     getter: spine.BoneData.prototype.getProp_color,
```

Comment:
Is this missing or what ?

Reply:
c++ class BoneData with no color property.